Gemfile

```diff
@@ -105,3 +105,5 @@ gem 'uglifier', '>=1.0.3'
 gem 'mini_racer', '~> 0.4.0'
 gem 'yui-compressor'
 #end
+
+gem "test_after_commit", "~> 1.2"
```

Comment:
Should we move this to the development/test group?

Reply:
Done!